packages/in_app_purchase/lib/src/billing_client_wrappers/billing_client_wrapper.dart

```diff
@@ -71,6 +73,17 @@ class BillingClient {
   Future<bool> isReady() async =>
       await channel.invokeMethod<bool>('BillingClient#isReady()');
 
+  /// Enable the [BillingClientWrapper] to handle pending purchases.
+  ///
+  /// This method is required to be called when initialize the application.
```

Comment:
Small optional wording nit:

```suggestion
  /// Play requires that you call this method when initializing your application.
```

https://developers.google.com/style/voice